test/jdk/javax/swing/regtesthelpers/JRobot.java

```diff
@@ -31,19 +42,10 @@
  * </pre>
  *
  */
-import java.awt.AWTException;
-import java.awt.Component;
-import java.awt.Dimension;
-import java.awt.EventQueue;
-import java.awt.Point;
-import java.awt.Rectangle;
-import java.awt.event.InputEvent;
-import java.awt.event.KeyEvent;
-import javax.swing.SwingUtilities;
 
 public class JRobot extends java.awt.Robot {
-    private static int DEFAULT_DELAY = 550;
-    private static int INTERNAL_DELAY = 250;
+    final private static int DEFAULT_DELAY = 550;
+    final private static int INTERNAL_DELAY = 250;
```

Comment:
Please use, [the *blessed* modifier order](https://docs.oracle.com/javase/8/docs/api/java/lang/reflect/Modifier.html#toString-int-): `private static final`.